changelog.md

```diff
@@ -8,6 +8,12 @@
 
 ### Fixed
 
+## [v0.1.1] - 2024-05-15
```

Comment:
Let's hyperlink this release version to the release page. That way, don't need to maintain a list of release links as the bottom